libcipher/transpo.py

```diff
@@ -0,0 +1,49 @@
+"""Transposition cipher implementation
+
+Provides functions to encrypt / decrypt a message by changing the
+character position of plain text.
+
+The Transposition decrypt function will simulate the columns and
+rows of the grid that the plain text is written on by using a list of
+strings. No of Columns is calculated by (length of the text)/key.
+key value is consider as no of rows.
+
+>>> decrypt(8, 'Cenoonommstmme oo snnio. s s c')
+'Common sense is not so common.'
+
+"""
+from __future__ import division
+import math
+
+
+def decrypt(key, message):
+    """Returns message decrypted using Transposition Cipher
+
+    Args:
+         key : Integer indicating Encryption key to decrypt message.
+         message : An encrypted string
```

Comment:
Why did we change this? Please see http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html for examples.